AH_Externals_Package/AHarker_Externals/docs/refpages/aharker-ref/schedulemessage.maxref.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version='1.0' encoding='utf-8' standalone='yes'?>
+<?xml-stylesheet href='./_c74_ref.xsl' type='text/xsl'?>
+<c74object name="schedulemessage" module="AHarker Externals" category="Threading">
+  <digest>
+    A generalised delay or pipe for any message type
+  </digest>
+  <description>
+      Schedule a message to happen at some point in the current timing thread (high priority when overdrive is on), otherwise all events are executed in a single low priority thread.
+  </description>
+  
+  <objarglist>
+      <objarg name="delay" optional="0" type="number">
```

Comment:
The optional="0" implies that the argument is required. This should only be used for arguments without which an object won't instantiate (for instance try putting [send] in a box with no name and see what happens in max.